src/simulator/services/resources/directory.py

```diff
@@ -8,7 +8,9 @@
 from simulator.services.debug import DebugLevel
 from simulator.services.service import Service
 from simulator.services.services import Services
+from simulator.services.resources.smart_unpickle import load as smart_load
 
+from typing import NamedTuple
```

Comment:
check if NamedTuple is actually needed here

Reply:
Removed.